var/spack/repos/builtin/packages/fckit/package.py

```diff
@@ -16,18 +16,12 @@ class Fckit(CMakePackage):
     git = "https://github.com/ecmwf/fckit.git"
     url = "https://github.com/ecmwf/fckit/archive/0.9.0.tar.gz"
 
-    maintainers = ['rhoneyager', 'climbfuji']
+    maintainers = ['climbfuji']
 
     version('master', branch='master')
     version('develop', branch='develop')
-    version('0.9.5', commit='7ec9cf8ad8b619a8319199e834171c11e111c888', preferred=True)
-    version('0.9.4', commit='a83cfe6d4ed22c954548dd7c31e1fbad3cd2f908')
-    version('0.9.3', commit='3f612e107682c61c0b6806ea3fc12e9509a90664')
-    version('0.9.2', commit='26439f09a421b29d745f4c4810d7d40f2820f5ec')
-    version('0.9.1', commit='0b2c04d29ff141d1963e21da2add2e70f01163ce')
-    version('0.9.0', commit='9cd993a524264e079ae260dbc89faea599e270fc')
-    version('0.8.0', commit='4cd749f1eeac64eece00adb50abd072ea14fa2b1')
-    version('0.7.0', commit='5a9ad884c087ae4c188a5937acf078514519778f')
```

Comment:
I think it's OK to remove these as in this PR, just checking if there are any possible drawbacks?

Reply:
No drawbacks, we never installed any of these old versions since we moved to spack-stack (and we created the fckit spack package ourselves, it doesn't exist in the authoritative repo yet)